app/assets/stylesheets/application.css.scss

```diff
@@ -98,6 +98,24 @@ input {
   margin-top: 1%;
 }
 
+.field > input[type="email"], input[type="password"] {
+  border-radius: 4px;
```

Comment:
Properties should be ordered border, border-radius, height, margin-bottom, margin-top, width